client/dom/shapes.js

```diff
@@ -0,0 +1,43 @@
+export const shapes = {
+	filledCircle: 'M 8,8 m 8,0 a 8,8 0 1,0 -16,0 a 8,8 0 1,0 16,0', //circle filled
```

Comment:
Making this a dict causes you to have to do Object.values and Object.keys, also you do this everytime you need to access by index